content/en/workshop/index.adoc

```diff
@@ -60,14 +61,14 @@ We welcome submissions on, but not limited to, the following topics:
 * Learning from sparse data: Novel methods for learning from small or sparse data in Turkic languages
 * Resources: Datasets, benchmarks, and software libraries for NLP models in Turkic languages
 
-Note: All deadlines are 11:59 pm UTC -12h (“anywhere on Earth”).
+Note: All deadlines are 11:59 pm UTC -12h ("anywhere on Earth").
 
 == "Hack Together" programming event
 
-In addition to the workshop itself, the second day will be devoted to a full-day collaborative programming event "Hack Together".
-Our goal is to demonstrate the SIGTURK NLP library and interested parties can contribute to the integration of new NLP methods and models into the SIGTURK pipeline.
+In addition to the workshop itself, the second day will contain presentations and collaborative events with a particular focus on coding guidelines and practices for behavior-memory safety and test-driven development.
+Our goal is to demonstrate the upbringing of the SIGTURK NLP library, and interested parties can contribute to the integration of new NLP methods and models or their building blocks into the SIGTURK pipeline.
```

Comment:
The term "upbringing" doesn't really make sense to me here. Maybe we can reword as "ongoing development" or something? If the intent is to communicate that the development will begin at the workshop, then maybe we can say "Our goal is to bootstrap the development of the SIGTURK library and encourage interested parties to contribute....", or something like that?

Reply:
@j0ma I meant the reasons and process that lead to the formation of core guidelines in a sense, and not the start of development at the workshop. I don't foresee any development to happen at workshop actually: just presentation and demo, maybe a little collaborative session. I'd prefer to keep the wording

Reply:
Thanks or the clarification; let's just keep this wording, especially in the interest of time.